src/core_modules/capture-core/components/Pages/MainPage/MainPage.container.js

```diff
@@ -120,6 +120,14 @@ const MainPageContainer = () => {
         showAllAccessible,
         categoryOptionIsInvalidForOrgUnit,
     });
+    const reduxSelectedTemplateId
+        = useSelector(({ workingListsTemplates }) => workingListsTemplates.teiList?.selectedTemplateId);
+    const workingListProgramId = useSelector(({ workingLists }) => workingLists
+        .teiList
+        ?.currentRequest
+        ?.queryParams
+        ?.program,
+    );
```

Comment:
Is it possible to calculate these values ​​along with the other selectors in `useSelectorMainPage`? Something like this:

```
const useSelectorMainPage = () =>
    useSelector(
        ({ currentSelections, activePage, workingListsTemplates, workingLists }) => ({
            reduxSelectedTemplateId: workingListsTemplates.teiList?.selectedTemplateId,
            workingListProgramId: workingLists.teiList?.currentRequest?.queryParams?.program,
            .....
        }),
        shallowEqual,
    );
```